site/src/main/paradox/migrations/v0.12.0-Migration-Guide.md

```diff
@@ -41,6 +47,12 @@ scoll.write(PubsubIO.string(topic, idAttribute, timestampAttribute))(PubsubIO.Wr
 scoll.write(PubsubIO.withAttributes[String](topic, idAttribute, timestampAttribute))(PubsubIO.WriteParam())
 ```
 
+Note: you can run the following sbt command to run the relevant [scalafix](https://scalacenter.github.io/scalafix/docs/developers/tutorial.html#run-the-rule-from-source-code) rules to update your code and/or get some guidance to fix the code: 
```

Comment:
```suggestion
Note: you can run the following sbt command to run the relevant [scalafix](https://scalacenter.github.io/scalafix/docs/developers/tutorial.html#run-the-rule-from-source-code) rules to automatically update deprecated Pub/Sub API usages: 
```

Reply:
well it's not always automatics updates, for lint it is just warning in the logs